### DISCUSSION TITLE
Gitbook - unable to edit on my phone

### DISCUSSION THREAD

**Discussion by GHK007** [GitbookIO/community#241] [0. General bugs and questions] [answered]
Hi,

Probably a very easy question but I cant find it.
I just started with making a gitbook. On the laptop it works perfectly.

However, I noticed, when I logging in via my phone I can edit the pages (text images etc?)

Any Idea what I am doing wrong?

Many thanks! :)

**Comment by addisonschultz** [comment, marked as answer]
Hi @GHK007! Thanks for raising this - it's something we're aware of! Unfortunately it isn't possible to edit GitBook on mobile or smaller sized screens at this point. It's something we'd like to look into in the future!

**Reply by GHK007** [reply]
Hi, Thank you for the reply!
Yes I do think this would be very helpfull. 
This would make it very low key/possible to make small adjust/additions during the day when something popsup in your mind during comuting, waiting or mini breaks.. :) (many small things create great things ;))

Looking forward to the update :)
Would it be possible to notify when it works?

Many thanks! :)

**Reply by paulsybrandy** [reply]
I second this. Was driving me nuts why I could not edit anything on my mobile away from home and my desktop PC. Please bring us mobile editing capabilities.

**Comment by maderlock** [comment]
This would be the killer Gitbook feature for me. I was planning to use this on the got to update rich notes, so that rules the tool out sadly. I tried forcing the screen to display the desktop site, use the phone in landscape, but it seems this has been done with a great deal of care to ensure that even if someone were willing to take the risks, Gitbook can't be used on a mobile.

**Comment by dangnelson** [comment]
Yeah I'm on the beach wishing I could make some documentation edits I was thinking of. This would be a great addition to the app.
